query-service-impl/src/main/java/org/hypertrace/core/query/service/projection/ProjectionTransformation.java

```diff
@@ -358,43 +358,26 @@ private QueryRequest rebuildRequest(
   private Filter rebuildFilterForComplexAttributeExpression(
       Filter originalFilter, List<OrderByExpression> orderBys) {
 
-    Filter filterFromOrderBys = createFilterForComplexAttributeExpressionFromOrderBy(orderBys);
     Filter updatedFilter = updateFilterForComplexAttributeExpressionFromFilter(originalFilter);
+    List<Filter> childFilterList = createFilterForComplexAttributeExpressionFromOrderBy(orderBys);
```

Comment:
nit: The naming here is a bit off - these aren't really children of anything. These are just the extra filters from the order by - later when we use them to construct the final filter, they may or may not become children (the use of the term "childFilter" throughout the PR has this same ambiguity).

Reply:
renamed to filterList